changelog-template.md

```diff
@@ -0,0 +1,41 @@
+# Change Log Template
```

Comment:
That's up to you. Because our "open-source-template" project is still private, I couldn't publicly refer it in the actual `CHANGELOG`. I'm happy to dump the template if you are; just let me know.